include/SearchForm/SearchForm2.php

```diff
@@ -186,7 +186,7 @@ public function display($header = true)
         $this->th->ss->assign('action', $this->action);
         $this->th->ss->assign('displayView', $this->displayView);
         $this->th->ss->assign('viewTab', $this->getViewTab());
-
+        $this->th->ss->assign('user_options', get_user_array(false)); // Fix https://github.com/salesagility/SuiteCRM/issues/10264
```

Comment:
I don't think we should fill Suitecrm with comments with each one of the issues.
That's what Git history is for.